### PR TITLE
Add blog post for week 23

### PR DESCRIPTION
**Pull request by nforro**

### CODE REVIEW

File: content/posts/weekly/June-2023.md [hunk 1]
```diff
@@ -8,3 +8,23 @@ weight: 51
...
+  ([packit-service#2080](https://github.com/packit/packit-service/pull/2080))
+- Fixed filters in `files_to_sync` not being applied properly.
+  ([packit#1977](https://github.com/packit/packit/pull/1977))
+- It is now possible to specify labels for a job and trigger tests based on labels
+  using a new `--labels` option of the `/packit test` command.
+  ([packit#1988](https://github.com/packit/packit/pull/1988),
+  [packit-service#2088](https://github.com/packit/packit-service/pull/2088))
+- We have finally fixed the issue that could've caused inconsistencies when tests
+  from a PR were supposed to be merged during the test runs on the Testing Farm.
+  ([packit-service#2090](https://github.com/packit/packit-service/pull/2090))
```
Comment (lachmanfrantisek) [inline content/posts/weekly/June-2023.md:30]: @mfocko Do we wan to phrase this like that? There was no issue on "our" side..;)

```suggestion
- The issue that could've caused inconsistencies when tests
  from a PR were supposed to be merged during the test runs on the Testing Farm has been fixed.
  ([packit-service#2090](https://github.com/packit/packit-service/pull/2090))
```

or 

```suggestion
- The Testing Farm team have fixed the issue that could've caused inconsistencies when tests
  from a PR were supposed to be merged during the test runs on the Testing Farm.
  ([packit-service#2090](https://github.com/packit/packit-service/pull/2090))
```

:
:

File: content/posts/weekly/June-2023.md [hunk 2]
```diff
@@ -8,3 +8,23 @@ weight: 51
...
+  ([packit-service#2079](https://github.com/packit/packit-service/pull/2079))
+- If the `vm_image_build job` doesn't have Copr project/owner configured,
+  Copr project used for the latest Copr build of the pull request will be now used.
+  ([packit-service#2080](https://github.com/packit/packit-service/pull/2080))
+- Fixed filters in `files_to_sync` not being applied properly.
+  ([packit#1977](https://github.com/packit/packit/pull/1977))
+- It is now possible to specify labels for a job and trigger tests based on labels
+  using a new `--labels` option of the `/packit test` command.
+  ([packit#1988](https://github.com/packit/packit/pull/1988),
+  [packit-service#2088](https://github.com/packit/packit-service/pull/2088))
```
Comment (lachmanfrantisek) [inline content/posts/weekly/June-2023.md:27]: I would maybe put both Strimzi PRs closer together or maybe merge them into one line:

```suggestion
- `/packit test` command has a new `--identifier` option. When this option is specified,
  Testing Farm will trigger only jobs with that identifier.
  ([packit-service#2079](https://github.com/packit/packit-service/pull/2079))
- It is now possible to specify labels for a job and trigger tests based on labels
  using a new `--labels` option of the `/packit test` command.
  ([packit#1988](https://github.com/packit/packit/pull/1988),
  [packit-service#2088](https://github.com/packit/packit-service/pull/2088))
- If the `vm_image_build job` doesn't have Copr project/owner configured,
  Copr project used for the latest Copr build of the pull request will be now used.
  ([packit-service#2080](https://github.com/packit/packit-service/pull/2080))
- Fixed filters in `files_to_sync` not being applied properly.
  ([packit#1977](https://github.com/packit/packit/pull/1977))
```

or

```suggestion
- `/packit test` command has two new options: With `--identifier` option,
  Testing Farm will trigger only jobs with that identifier.
  ([packit-service#2079](https://github.com/packit/packit-service/pull/2079))
  Also, it is now possible to specify labels for a job and trigger tests based on labels
  using a `--labels` option of the `/packit test` command.
  ([packit#1988](https://github.com/packit/packit/pull/1988),
  [packit-service#2088](https://github.com/packit/packit-service/pull/2088))
- If the `vm_image_build job` doesn't have Copr project/owner configured,
  Copr project used for the latest Copr build of the pull request will be now used.
  ([packit-service#2080](https://github.com/packit/packit-service/pull/2080))
- Fixed filters in `files_to_sync` not being applied properly.
  ([packit#1977](https://github.com/packit/packit/pull/1977))
- 
```

File: content/posts/weekly/June-2023.md (hunk 1 above)
Comment (mfocko) [reply content/posts/weekly/June-2023.md:30]: I guess the first suggestion makes more sense, no one to blame :wink: :smile:

**Comment by nforro** [comment]
Thanks for the suggestions @lachmanfrantisek :pray:

**Review by lachmanfrantisek — APPROVED** [review]

**Review by lbarcziova — APPROVED** [review]
great week!

**Pull request merged**